packages/manager/modules/pci/src/projects/project/storages/containers/container/container.controller.js

```diff
@@ -10,6 +10,7 @@ export default class PciStoragesContainersContainerController {
     $window,
     CucCloudMessage,
     PciProjectStorageContainersService,
+    $stateParams,
```

Comment:
`$stateParams` is deprecated, so you should use `$transition.params().defaultCriteria` into the routing (into packages/manager/modules/pci/src/projects/project/storages/objects/object/object.routing.js), and pass it into component bindings

Reply:
Good point, I had no idea about the deprecated status of $stateParams. I made the changes as you suggested, let me know if something is not as expected.